target/linux/mpc85xx/p1020/config-default

```diff
@@ -8,6 +8,7 @@ CONFIG_EEPROM_LEGACY=y
 CONFIG_GENERIC_CLOCKEVENTS_BROADCAST=y
 CONFIG_GENERIC_TBSYNC=y
 # CONFIG_GPIO_MAX77620 is not set
+CONFIG_GPIO_74X164=y
```

Comment:
This should go above `CONFIG_GPIO_MAX77620`.

Reply:
Fixed in force-push 557192907b9652b21f1cc8890b807edae22aed40.